src/main/webapp/app/entities/exercise-hint/exercise-hint.component.html

```diff
@@ -0,0 +1,63 @@
+                Create new Exercise Hint
+            </span>
+        </button>
+    </h2>
+    <jhi-alert></jhi-alert>
+    <br />
+    <div class="alert alert-warning" *ngIf="exerciseHints?.length === 0">
+        <span jhiTranslate="artemisApp.exerciseHint.home.notFound">No exerciseHints found</span>
+    </div>
+    <div class="table-responsive" *ngIf="exerciseHints?.length > 0">
+        <table class="table table-striped">
+            <thead>
+                <tr>
+                    <th><span jhiTranslate="global.field.id">ID</span></th>
+                    <th><span jhiTranslate="artemisApp.exerciseHint.title">Title</span></th>
+                    <th><span jhiTranslate="artemisApp.exerciseHint.content">Content</span></th>
+                    <th><span jhiTranslate="artemisApp.exerciseHint.exercise">Exercise</span></th>
+                    <th></th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr *ngFor="let exerciseHint of exerciseHints; trackBy: trackId">
+                    <td>
+                        <a [routerLink]="['/exercise', exerciseId, 'hints', exerciseHint.id, 'view']">{{ exerciseHint.id }}</a>
+                    </td>
+                    <td>{{ exerciseHint.title }}</td>
+                    <td [innerHTML]="exerciseHint.content | htmlForMarkdown"></td>
+                    <td>
+                        <span>{{ exerciseHint.exercise?.id }}</span>
+                    </td>
+                    <td class="text-right">
+                        <div class="btn-group">
+                            <button type="submit" [routerLink]="['/exercise', exerciseId, 'hints', exerciseHint.id, 'view']" class="btn btn-info btn-sm">
+                                <fa-icon [icon]="'eye'"></fa-icon>
+                                <span class="d-none d-md-inline" jhiTranslate="entity.action.view">View</span>
+                            </button>
+                            <button type="submit" [routerLink]="['/exercise', exerciseId, 'hints', exerciseHint.id, 'edit']" class="btn btn-primary btn-sm">
+                                <fa-icon [icon]="'pencil-alt'"></fa-icon>
+                                <span class="d-none d-md-inline" jhiTranslate="entity.action.edit">Edit</span>
+                            </button>
+                            <button
+                                type="submit"
+                                [routerLink]="['/', { outlets: { popup: ['exercise', exerciseId, 'hints', exerciseHint.id, 'delete'] } }]"
+                                replaceUrl="true"
+                                queryParamsHandling="merge"
+                                class="btn btn-danger btn-sm"
+                            >
+                                <fa-icon [icon]="'times'"></fa-icon>
+                                <span class="d-none d-md-inline" jhiTranslate="entity.action.delete">Delete</span>
```

Comment:
Please add some space between the buttons. The same way we have it on course exercises page.

Reply:
You are right, I wondered why it looked weird here.